README.md

```diff
@@ -6,13 +6,14 @@
 
 ```bash
 git clone https://github.com/opencontainers/runc
```

Comment:
yea, I thought about modifying that part too in this PR but I didn't want to overlap with #14.
I can do it if people want though....